### PR TITLE
Registration confirmation fields

### PR DESCRIPTION
**Pull request by mschnitzer**
screenshot at 2018-01-22 12-49-10

### CODE REVIEW

**Comment by mschnitzer** [comment]
[Provide a "repeat password" field during registration](https://trello.com/c/7E3GaDsD/282-provide-a-repeat-password-field-during-registration)

**Comment by mschnitzer** [comment]
@hennevogel 

What do you mean with your question?

Are you asking the team if we have had any problems without an additional field or are you asking if the team has any problems when we merge this?

If you ask if we had problems with not having an additional field in the past: I don't think we've received any issues from our customers/users, but typos happen and especially when it comes to passwords, as you don't see what you're typing. I don't see any disadvantage in having this as many projects have a confirmation field and I even think the most of the projects have it. I think we didn't receive any issues yet, because:

a) We don't use the registration field on build.opensuse.org
b) Users that register on external OBS platforms, might just contact their administrator or create a new account and I doubt that most of them are going to create an issue here.

I think it's a nice to have and there was even a PR opened months ago and this was a dangling task in our fruit-tree-column (since March 7, 2017)

**Comment by bgeuken** [comment]
:+1: for the password verification field. For the email field I am indifferent. As a user I am always annoyed by these fields, because they force me to c&p the email address.
And in OBS one can change his address in case it was added incorrectly. At least in the non-proxy mode, and in proxy mode this seems to be configurable via the proxy.

tldr;

I like the password verification field, I would drop the email verification field.

**Comment by adrianschroeter** [comment]
On Freitag, 19. Januar 2018, 16:33:19 CET wrote Björn Geuken:
> :+1: for the password verification field. For the email field I am indifferent. As a user I am always annoyed by these fields, because they force me to c&p the email address.
> And in OBS one can change his address in case it was added incorrectly. At least in the non-proxy mode, and in proxy mode this seems to be configurable via the proxy.
> 
> tldr;
> 
> I like the password verification field, I would drop the email verification field.

yep! +1

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imendörffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nürnberg) 
 
Maxfeldstraße 5                         
90409 Nürnberg 
Germany

**Comment by mschnitzer** [comment]
OK, I'll gonna remove the email verification field, as there were already 3 people against it.

**Comment by mschnitzer** [comment]
@bgeuken @adrianschroeter I've updated the code and the screenshot.
